services/movie.py

```diff
@@ -6,6 +7,7 @@
 def get_movies(
     genres_ids: list[int] = None,
```

Comment:
Using `list[int]` as a default argument value can be problematic because lists are mutable. It's better to use `None` as a default value and then initialize the list inside the function if needed. This avoids potential issues with shared state between function calls.